docs/src/understand/thin-edge-json.md

```diff
@@ -114,44 +169,59 @@ and hence must not be used as measurement keys:
-Any data can be forwarded to the connected cloud-provider as well, by publishing the data to some standard topics.
-
-All topics with the prefix `tedge/` are reserved by `thin-edge.io` for this purpose.
-To send measurements to `thin-edge.io`, the measurements represented in Thin Edge JSON format can be published 
-to the `tedge/measurements` topic.
-Other processes running on the thin-edge device can subscribe to this topic to process these measurements.
-
-If the messages published to this `tedge/measurements` topic is not a well-formed Thin Edge JSON, 
-then that message won’t be processed by `thin-edge.io`, not even partially,
-and an appropriate error message on why the validation failed will be published to a dedicated `tedge/errors` topic.
-The messages published to this topic will be highly verbose error messages and can be used for any debugging during development.
-You should not rely on the structure of these error messages to automate any actions as they are purely textual data 
-and bound to change from time-to-time.
+## Events
 
-More topics will be added under the `tedge/` topic in future to support more data types like events, alarms etc.
-So, it is advised to avoid any sub-topics under `tedge/` for any other data exchange between processes.
+*Events* are notifications that something happened on the device, its environment, the domain application or the software system.
+For instance:
+- a door has been closed
+- a process started
+- a user has started an ssh session
 
-Here is the complete list of topics reserved by `thin-edge.io` for its internal working:
+```sh te2mqtt
+tedge mqtt pub /tedge/events/login '{
+  "text": "A user just logged in",     // 'text' message of that event
+  "time": "2021-01-01T05:30:45+00:00"  // optional 'timestamp' of that event
+  "someOtherCustomFragment": {         // free 'custom fragments'
+    "nested": {
+      "value": "extra info"
+    }
+  }
+}'
+```
 
-| Topic | Description |
-| --- | --- |
-| `tedge/` | Reserved root topic of `thin-edge.io` |
-| `tedge/measurements` | Topic to publish measurements to `thin-edge.io` |
-| `tedge/measurements/<child-id>` | Topic to publish measurements to `thin-edge.io`'s child device |
-| `tedge/errors` | Topic to subscribe to receive any error messages emitted by `thin-edge.io` while processing measurements|
+|Reference           | Description                                                                                                                                                                                |
+| ------------------ |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|`event-type`        | a string part of the MQTT topic, that identifies the event uniquely in context of the device                                                                                               |
```

Comment:
Using an underscore `_` ensures the work does not get split over two lines.

```suggestion
|`event_type`        | a string part of the MQTT topic, that identifies the event uniquely in context of the device                                                                                               |
```

Reply:
Updated